<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_s3</class>
 <widget class="QDialog" name="update_s3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemNo</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_itemNo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_itemNo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Check ItemNo</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_stock">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_status">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_price">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stock</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
